### PR TITLE
Inception reflections

### PR DESCRIPTION
**Pull request by calebschoepp**
Here is a new blog post about my reflections on the Inception Studio weekend.

### CODE REVIEW

File: content/blog/inception_studio_weekend_reflections.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+**TODO fix publish date**
+
+I recently spent a weekend in California at an Inception Studio event. For three days I hacked on large language model (LLM) ideas with some really smart people. I learned so much from the experience that I wanted to write down some of my biggest learnings.
+
+## Life
+
+### Take more risk
+
+At this time in my life I should take as much risk as I can fathom. Taking large risks early in life means you have a longer time to compound on the successful outcomes of these risks. I also have multiple factors that make me very resilient to failure:
```
Comment (calebschoepp) [inline content/blog/inception_studio_weekend_reflections.md:15]: You did a typo here. Try to do better.

File: content/blog/inception_studio_weekend_reflections.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+When I asked one of the attendees for advice on being an engineer manager I was given some advice that I thought was very profound.
+
+Imagine that one of your direct reports is doing something problematic. In your gut you know you should have a conversation with them and bring up the issue. However, this requires some confrontation and isn't easy to do. So you take the easy way out and don't say anything. Naturally the problem persists and gets worse over time. As months pass, the problem grows into something that can no longer be ignored. You need to fire this employee because of the problem. When you go to fire them they of course ask you why. You explain that, "it is because of the problem", to which they say, "why didn't you tell me about the problem?!" No one is happy.
+
+The point of the story is that you have a moral imperative as a manager to have these difficult conversations early. Not doing so is just setting up everyone for more pain and difficulty.
+
+### A mental model for entrepreneurship
+
+Only two things matter in entrepreneurship: product market fit and founder product fit.
```
Comment (calebschoepp) [inline content/blog/inception_studio_weekend_reflections.md:50]: Another comment

**Review by calebschoepp — COMMENTED** [review]
Overall I think you're a great son caleb

File: content/blog/inception_studio_weekend_reflections.md (hunk 1 above)
Comment (calebschoepp) [reply content/blog/inception_studio_weekend_reflections.md:15]: Oh thanks I'll fix that.

File: content/blog/inception_studio_weekend_reflections.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
+---
+title: "Inception Studio Weekend Reflections"
```
Comment (PaulSchoepp) [inline content/blog/inception_studio_weekend_reflections.md:2]: Caleb

File: content/blog/inception_studio_weekend_reflections.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+draft: false
+---
+
+**TODO fix publish date**
+
+I recently spent a weekend in California at an Inception Studio event. For three days I hacked on large language model (LLM) ideas with some really smart people. I learned so much from the experience that I wanted to write down some of my biggest learnings.
+
+## Life
+
+### Take more risk
```
Comment (PaulSchoepp) [inline content/blog/inception_studio_weekend_reflections.md:13]: little comment

File: content/blog/inception_studio_weekend_reflections.md (hunk 3 above)
Comment (calebschoepp) [reply content/blog/inception_studio_weekend_reflections.md:2]: Hi

File: content/blog/todo.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+draft: false
+---
+
+**TODO fix title**
+
+**TODO fix publish date**
+
+**TODO fix filename**
+
+I was recently at inception and I wanted to write down everything I learned.
```
Comment (PaulSchoepp) [inline content/blog/todo.md]: Define inception?

Comment (PaulSchoepp) [reply content/blog/todo.md]: Give overview.  I have xx take aways from the weekend...

File: content/blog/todo.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+### I should take as much risk as possible
+
+Ian and Greg. Worst case scenario is not bad at all.
```
Comment (PaulSchoepp) [inline content/blog/todo.md]: Hmmm--Ii don't think this is the thing im supposed to be edting.  Let me try again.....

File: content/blog/todo.md [hunk 7]
```diff
@@ -10,15 +10,42 @@ draft: false
 
 **TODO fix filename**
 
-I was recently at inception and I wanted to write down everything I learned.
+I recently spent a weekend in California at an Inception Studio event. For three days I hacked on large language model (LLM) ideas with some really smart people. I learned so much from the experience that I wanted to write down some of my biggest learnings.
```
Comment (PaulSchoepp) [inline content/blog/todo.md]: Is Inception Studio Event widely understood or should you define?

Comment (PaulSchoepp) [reply content/blog/todo.md]: Give a preview of article.  
I wanted to share  by big take aways:  Risk more; LLM insights; etc

File: content/blog/todo.md [hunk 8]
```diff
@@ -10,15 +10,42 @@ draft: false
...
+## LLM's
 
-Ian and Greg. Worst case scenario is not bad at all.
+### New mental models for LLM's
+
+Each of these mental models are too simple but this is exactly what makes them useful shorthands for understanding LLM's.
+
+An LLM is a compression function run across the entire internet. When you ask GPT-3 a question you're really just asking that question to the mean of the internet.
+
+An LLM is like an axe. The size of the axe head is the size of the model; GPT-2 is a hatchet; GPT-3 is a standard axe. Fine tuning is like sharpening your axe. How you prompt the LLM is like how you wield your axe. As you try to chop through tougher wood (thornier problems) you either need a bigger axe head, a sharper blade, or better technique -- or all of the above.
```
Comment (PaulSchoepp) [inline content/blog/todo.md]: It's good that you have your axe license to make these comparisons

File: content/blog/todo.md [hunk 9]
```diff
@@ -10,15 +10,42 @@ draft: false
...
+
+An LLM is like an alien species who has landed on earth and we can ask questions of. They are vastly more intelligent than us. But, they are also communicate in a completely different paradigm. If we spend time learning how to ask them the right questions we can unlock their vast intelligence.
```
Comment (PaulSchoepp) [inline content/blog/todo.md]: don't end sentence with "of"

File: content/blog/todo.md [hunk 10]
```diff
@@ -10,15 +10,42 @@ draft: false
...
 
-### I should see a therapist or lifecoach regularly
+### The future of LLM's is very exciting
 
-You spend all day in your brain as a programmer and mental health is incredibly important.
+This weekend has gotten me very excited for the future of LLM's. I didn't pay much attention when GPT-3 came out but you can be sure that I'll be on the edge of my seat for the release of GPT-4. I think that it is going to display some emergent properties that are hard to fathom right now. Another way to think about this is that we are somewhere near the start of an innovation s-curve -- a very exciting place to be indeed. I have lots of questions about the future of LLM's that I look forward to getting answers to:
```
Comment (PaulSchoepp) [inline content/blog/todo.md]: don't end sentence with "to"

File: content/blog/todo.md [hunk 11]
```diff
@@ -28,37 +55,40 @@ Because if you don't have it you're going to end up firing them in a year and a
...
+Ultimately when I'm building something the goal is product market fit. Remembering this feeling of being pulled vs. pushed will help me know when I've found product market fit and when I should keep looking.
 
-I want to learn more about this.
+## Life
 
-### I want to do more hackathons
+### The value of regular therapy or life coaching
 
-I want to do more of this and meet a lot more smart motivated people.
+I've seen therapists in acute contexts and it has been transformational. Correspondingly I've become a big advocate for therapy. However, I've always tied it to acute issues in my life. What if I integrated it more regularly into my life? This could be hugely beneficial since a programmer spends most of the day inside their own head. Speaking from personal experience I can say that dips in my mental health can wreak havoc on my productivity (not mentioning its effects on the rest of my life). It only makes sense that mental health should be regularly cared for like physical health.
```
Comment (PaulSchoepp) [inline content/blog/todo.md]: Plural--programmer(s)      their head

File: content/blog/todo.md [hunk 12]
```diff
@@ -28,37 +55,40 @@ Because if you don't have it you're going to end up firing them in a year and a
...
 ### The importance of ego and humbleness
 
 Stay low ego
 
+---
+
+_Special thanks to Andy, Ian, Greg, Brian, and Oliver for some of these insights._
+
 Andy Chou thought I was a sr. engineer
+
```
Comment (PaulSchoepp) [inline content/blog/todo.md]: I'm pretty sure this is also not the right place to be editing.  
I might need a personal tutorial....
